mp/importer/batcher.py

```diff
@@ -26,9 +26,73 @@ def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=
             if position_in_batch == 0: 							# ends completed batches
                 end_batch_callback()
                 ended = True
-        if max_batches is not None and current_batch == batch_start + max_batches:					# if gets maximum allowed batches, break 
+        if max_batches is not None and current_batch == batch_start + max_batches:	# if gets maximum allowed batches, break 
             break
     if ended is False:
         end_batch_callback()	
 
 
+def new_parse_arguments(parser):		 						
+    """ Adds batcher arguments to the parser									 
+
+    This function gets the parser containing user inputs
+    It adds arguments and returns a parser
+    The parsing of args is performed in the main script 
+    """ 
+    parser.add_argument('--argument_1', dest='batcher_argument_1', type=int, help="1st argument")
```

Comment:
I'm confused why you don't just have the real arguments here? i.e. why --argument_1 and not --batch-size?